shell/common/common_hive.sh

```diff
@@ -213,13 +214,14 @@ get_hive_substitutions() {
     HIVE_BYTES_PER_REDUCER "$HIVE_BYTES_PER_REDUCER" \
     PARQUET_COMPRESSION "$PARQUET_COMPRESSION" \
     ORC_COMPRESSION "$ORC_COMPRESSION" \
-    EXPERIMENT_ID "$EXPERIMENT_ID"
+    EXPERIMENT_ID "$EXPERIMENT_ID,$1"
```

Comment:
This isn't the experiment ID any more - I would suggest changing the template variable to `##APPLICATION_TAGS##` and then the second argument on this line would be something like `"$EXPERIMENT_ID,$display_name"`.